address.go

```diff
@@ -261,8 +263,14 @@ func (a *Address) IsValid(chain ChainID) bool {
 // invalid code-words in the [64,45] code
 // these constants are used to generate non-Flow-Mainnet addresses
 const (
-	invalidCodeTestnet  = uint64(0x6834ba37b3980209)
-	invalidCodeEmulator = uint64(0x1cb159857af02018)
+	// invalidCodeTestNetwork is the invalid codeword used for long-lived test networks.
```

Comment:
Should we keep a cross-note to keep them in sync with `model/flow/address.go` in flow-go?